modules/AOR_Reports/AOR_Report.php

```diff
@@ -1324,7 +1324,40 @@ function build_report_query_where($query = array()){
                         $value = "{$value} OR {$field} IS NULL";
                     }
 
-                    if (!$where_set) $query['where'][] = ($tiltLogicOp ? '' : ($condition->logic_op ? $condition->logic_op . ' ': 'AND ')) . $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                    if(!$where_set) {
+                        if ($condition->value_type == "Period") {
+                            $date = '"' . getPeriodEndDate($params)->format('Y-m-d H:i:s') . '"';
+                            if (array_key_exists($condition->value, $app_list_strings['date_time_period_list'])) {
+                                $params = $condition->value;
+                            } else {
+                                $params = base64_decode($condition->value);
+                            }
+                            $value = '"' . getPeriodDate($params)->format('Y-m-d H:i:s') . '"';
+
+                            switch ($app_list_strings['aor_sql_operator_list'][$condition->operator]) {
+                                case "=":
+                                    $query['where'][] = $field . ' BETWEEN ' . $value .  ' AND ' . '"' . $date . '"';
+                                    break;
+                                case "!=":
+                                    $query['where'][] = $field . ' NOT BETWEEN ' . $value .  ' AND ' . '"' . $date . '"';
+                                    break;
+                                case ">":
+                                    $query['where'][] = $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                                    break;
+                                case "<":
+                                    $query['where'][] = $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                                    break;
+                                case ">=":
+                                    $query['where'][] = $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                                    break;
+                                case "<=":
+                                    $query['where'][] = $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                                    break;
+                            }
+                        } else {
+                            $query['where'][] = ($tiltLogicOp ? '' : ($condition->logic_op ? $condition->logic_op . ' ': 'AND ')) . $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
```

Comment:
do the previous factor in the $tiltLogicOp check, should maybe be a more generic check if within !where_set